test/snowcrashtest.h

```diff
@@ -146,6 +146,27 @@ namespace snowcrashtest {
             return signature;
         }
     };
+
+    /**
+     * \brief Helper to test source maps
+     */
+    struct SourceMapHelper {
+
+        /**
+         * If 'nth' is not given, sourceMap is of size 1 and checks the 0th element
```

Comment:
The point is. This should be clear from the name of the function helper and the comment. Which it is not.